release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -232,18 +232,6 @@ For more information, see xref:../security/encrypting-etcd.adoc#etcd-encryption-
 [id="ocp-4-13-auth"]
 === Authentication and authorization
 
-[id="ocp-4-13-psa-tp-feature-set"]
```

Comment:
@JoeAldinger I could use some RN advice here.

A future 4.12 z-stream is going to be adding this PSA restricted enforcement as a TP feature. So that's why I'm removing it from the 4.13 announcements (because it will technically come out with the z-stream first).

However, I had added it to the TP tables below. But I'm not sure how to modify that for this situation. Right now, it has it listed as Not Available in 4.12, and then TP in 4.13. But now it's going to be TP in 4.12, but only as of a particular 4.12.z or later.

Should I set the 4.12 one to "Technology Preview", or leave it as "Not Available"? Or I could set it to TP and add a footnote that says as of whichever 4.12.z it goes out with. wdyt?

Reply:
There is precedent for the last option "Set it to TP and add a footnote that says as of whichever 4.12.z it ships with." See `Pod-level bonding for secondary networks` in 4.12's table and asynchronous notes.

Reply:
I guess a followup question then would be should I update the 4.12 release notes table to add this then too, once the z-stream goes out?

Reply:
Yeah, when this ships one PR should be merged into 4.12 with the changes notating the specific 4.12.z and then another PR goes into 4.13 to update the table for consistency with 4.12. Does that make sense?